packages/sonataflow-deployment-webapp/src/components/validateWorkflowData.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import isPlainObject from "lodash/isPlainObject";
+
+export function validateWorkflowData(data: string): boolean {
```

Comment:
Not sure if this file belongs to the `components` folder.

Reply:
The original kogito-apps's path is
`kogito-apps/ui-packages/packages/workflow-form/src/envelope/components/WorkflowForm`  

I reused similar paths not to make things confusing, but I'm available to change this

Reply:
In this case, let's keep it that way. 
We can reevaluate this scenario when starting to reuse components from kogito-apps.